src/toir.c

```diff
@@ -422,7 +423,17 @@ int intrinsic_op(FuncDeclaration *fd)
         "4simd6__simdFNaNbNiNfE4core4simd3XMMdZNhG16v",
         "4simd6__simdFNaNbNiNfE4core4simd3XMMfZNhG16v",
         "4simd9__simd_ibFNaNbNiNfE4core4simd3XMMNhG16vhZNhG16v",
-
+#else
+        "4simd10__simd_stoFNaNbNiNfE2$7$3XMMNhG16v%4cZ%4c",
+        "4simd10__simd_stoFNaNbNiNfE2$7$3XMMdNhG16vZ%4d",
+        "4simd10__simd_stoFNaNbNiNfE2$7$3XMMfNhG16vZ%4d",
+        "4simd6__simdFNaNbNiNfE2$7$3XMMNhG16v%3hZ%3h",
+        "4simd6__simdFNaNbNiNfE2$7$3XMMNhG16v%3hhZ%3h",
+        "4simd6__simdFNaNbNiNfE2$7$3XMMNhG16vZ%3h",
```

Comment:
> how can it do backrefs here ?
> AFAICS it looses the ´core´

these strings are assumed to be prefixed by `_D4core`, see the comparisons below.